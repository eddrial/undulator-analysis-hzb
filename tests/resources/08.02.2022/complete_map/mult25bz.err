 error report for the data set mult25bz.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the derivative at  0.43020E+03 and  0.51386E+03 is zero 
 the derivative at  0.51386E+03 and  0.10746E+04 is zero 
 the derivative at  0.11025E+04 and  0.14671E+04 is zero 
 the derivative at  0.14671E+04 and  0.18563E+04 is zero 
 the derivative at  0.18563E+04 and  0.21060E+04 is zero 
 the rms value of the residuals is  0.16051E-02
 the maximum residual is  0.46482E-02
 the rms value of the lambda/2-filtered data is  0.15243E-02
 the maximum value of the lambda/2-filtered data is  0.13212E-01
