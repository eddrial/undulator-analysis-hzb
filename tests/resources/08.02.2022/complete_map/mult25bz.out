mult25bz.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -6.2350333E-03
   2  1.3194278E-05
   3 -6.2987456E-05
   4 -6.2982358E-06
   5 -3.5976923E-06
   6  5.5959595E+01
   7  1.0463722E+03
   8  1.0464290E+03
   9  1.0414152E+03
  10  2.6167891E+00
  11  1.4279954E+00
 number of iterations:   19
     sumsq:  4.09879E-03
       rms:  1.60507E-03  dimension of magnetic field
       rms:  2.57426E+01  percent of b_eff
   rms_ext:  9.12332E+00  percent of b_eff
 B_eff =   6.2350E-03
 B_max =  -6.1604E-03
 K_eff =    0.033
 lamda_1 (nm) =      2.54163

        eigenvectors in parameter space

 error                         direction
    15.71,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
    12.84,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00 -0.01  0.00
    11.54,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.01  0.00  1.00  0.00
     1.62,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.09,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00,  0.59  0.39 -0.11  0.50 -0.49  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.76 -0.03  0.27 -0.40  0.44  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.18  0.85 -0.21 -0.06  0.45  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.15  0.08  0.80  0.52  0.24  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.13 -0.35 -0.47  0.57  0.56  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.22956 Tmm
 kick_2:   0.22956 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.97767E+01
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.16051E-02
 the maximum residual is  0.46482E-02
      
 the rms value of the lambda/2-filtered data is  0.15243E-02
 the maximum value of the lambda/2-filtered data is  0.13212E-01
      
 the rms phase error (all data) is    1.355 deg.
 the maximum phase error (all data) is    3.319 deg.
      
 the rms phase error (extrema) is    5.545 deg.
 the maximum phase error (extrema) is    9.435 deg.
      
 the Walker-rms phase error (all data) is    1.354 deg.
 the Walker-maximum phase error (all data) is    3.231 deg.
      
 the Walker-rms phase error (extrema) is    1.379 deg.
 the Walker-maximum phase error (extrema) is    1.818 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             1.354736    
 local phase error            0.6522275    
 slowly varying phase error  NaN           
 oscillatory phase error     NaN           
 -----------------------------------------
